app/helpers/helpers.py

```diff
@@ -255,76 +269,81 @@ def send_notification(user, action, title, message):
+        message = notif['message'].format(
+            role_name=role_name,
+            event_name=event_name,
+            accept_link=accept_link,
+            decline_link=decline_link
+        )
+
+        send_notification(user, action, title, message)
 
 
 def send_notif_new_session_organizer(user, event_name, link):
-    notif = NOTIFS[NOTIF_NEW_SESSION]
-    action = NOTIF_NEW_SESSION
-    title = notif['title'].format(event_name=event_name)
-    message = notif['message'].format(event_name=event_name, link=link)
+    if MessageSettings.query.filter_by(action=NOTIF_NEW_SESSION).first().notif_status == 1:
+        notif = NOTIFS[NOTIF_NEW_SESSION]
+        action = NOTIF_NEW_SESSION
+        title = notif['title'].format(event_name=event_name)
+        message = notif['message'].format(event_name=event_name, link=link)
 
-    send_notification(user, action, title, message)
+        send_notification(user, action, title, message)
 
 
 def send_notif_session_schedule(user, session_name, link):
-    notif = NOTIFS[NOTIF_SESSION_SCHEDULE]
-    action = NOTIF_SESSION_SCHEDULE
-    title = notif['title'].format(session_name=session_name)
-    message = notif['message'].format(session_name=session_name, link=link)
+    if MessageSettings.query.filter_by(action=NOTIF_SESSION_SCHEDULE).first().notif_status == 1:
+        notif = NOTIFS[NOTIF_SESSION_SCHEDULE]
+        action = NOTIF_SESSION_SCHEDULE
+        title = notif['title'].format(session_name=session_name)
+        message = notif['message'].format(session_name=session_name, link=link)
 
-    send_notification(user, action, title, message)
+        send_notification(user, action, title, message)
 
 
 def send_notif_next_event(user, event_name, up_coming_events, link):
-    notif = NOTIFS[NOTIF_NEXT_EVENT]
-    action = NOTIF_NEXT_EVENT
-    title = notif['title'].format(event_name=event_name)
-    message = notif['message'].format(up_coming_events=up_coming_events,
-                                      link=link)
-
-    send_notification(user, action, title, message)
-
-
-def send_notif_session_accept_reject(user, session_name, acceptance,
-        link):
-    notif = NOTIFS[NOTIF_SESSION_ACCEPT_REJECT]
-    action = NOTIF_SESSION_ACCEPT_REJECT
-    title = notif['title'].format(session_name=session_name,
-                                  acceptance=acceptance)
-    message = notif['message'].format(
-        session_name=session_name,
-        acceptance=acceptance,
-        link=link
-    )
+    if MessageSettings.query.filter_by(action=NOTIF_NEXT_EVENT).first().notif_status == 1:
+        notif = NOTIFS[NOTIF_NEXT_EVENT]
+        action = NOTIF_NEXT_EVENT
+        title = notif['title'].format(event_name=event_name)
+        message = notif['message'].format(up_coming_events=up_coming_events,
+                                          link=link)
+
+        send_notification(user, action, title, message)
+
+
+def send_notif_session_accept_reject(user, session_name, acceptance, link):
```

Comment:
Right now we are following snake_case convention instead of camelCase convention